backend/gateway/meta/meta.go

```diff
@@ -119,6 +119,77 @@ func ResourceNames(pb proto.Message) []*auditv1.Resource {
 	return nil
 }
 
+// ExtractProtoPatternsValues takes a proto and returns its pattern populated with values
+func ExtractProtoPatternsValues(pb proto.Message) string {
+	m := pb.ProtoReflect()
+	opts := m.Descriptor().Options().ProtoReflect()
+
+	populatedPattern := []string{}
+
+	if opts.Has(identifierTypeDescriptor) {
+		v := opts.Get(identifierTypeDescriptor)
+		id := v.Message().Interface().(*apiv1.Identifier)
+
+		for _, pattern := range id.Patterns {
```

Comment:
Check support for more than one pattern, currently I believe its only 1.

Reply:
@danielhochman is it safe to assume that as it currently stands you can only have one pattern?

Reply:
talked in person and we only support one pattern for now, left a comment below as to why we assume there is only one item in the list.